src/jace/translator/jaxpr_translator_builder.py

```diff
@@ -0,0 +1,828 @@
+# JaCe - JAX Just-In-Time compilation using DaCe (Data Centric Parallel Programming)
+#
+# Copyright (c) 2024, ETH Zurich
+# All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+"""Contains the translator that actually builds an SDFG based on a Jaxpr description."""
+
+from __future__ import annotations
+
+import copy
+from collections.abc import Mapping, MutableSequence, Sequence
+from typing import TYPE_CHECKING, Any, Literal, cast, overload
+
+import dace
+from dace import data as ddata, properties as dprop
+from jax import core as jax_core
+
+
+if TYPE_CHECKING:
+    from jace import translator
+
+from jace import util
```

Comment:
I tried it and it does it not.
It probably wrote it in this order, because I tought ruff would put it in the correct order, but it does not look like that.
So much for automation.